dashboard/UI-Test/selenium_drivers/README.md

```diff
@@ -0,0 +1,9 @@
+Driver version currently shipped:
+- Chrome/Chromium = v105.0.5195.19
+- Firefox(gecko) = v0.31.0
+
+> NOTE:
+- Selenium_drivers folders have your selenium webdrivers geckodrivers(for firefox) and chromedrivers(for chrome and chromium)
+- If script fails due to drivers issue, you need to find appropriate selenium webdriver according to your browser version(choose latest if your browser is upto to date)
+  - [firefox](https://github.com/mozilla/geckodriver/releases) & [chrome/chromium](https://chromedriver.chromium.org/downloads)
```

Comment:
There is no need to have these references in their own list of _one_ bullet.  Just append them to the previous line.